release-process.rst

```diff
@@ -64,65 +79,158 @@ Example time line with only one major version at a time
             |     |     |     |     |     |     |     |     |     |     |     |     |
-    5.3     +++++++++++++-----D
-    5.4     |*****+++++++++++++++++++++++++-----------D
-    5.5     |     |     |******++++++++++++++++++++++++-----------D
-    5.6     |     |     |     |     |******++++++++++++++++++++++++-----------D
-    6.0     |     |     |     |                 |******++++++++++++++++++++++++-----------D
-    6.1     |     |     |     |                             |******++++++++++++++++++++++++-----------D
+    8.1     |++++++++++-------------------------D
+    8.2     |+++++++++++++++++++++++------------------------D
+    8.3     |     *****G++++++++++++++++++++++++------------------------D
+    8.4     |     |     |     |****G++++++++++++++++++++++++------------------------D
 
-Example time line with two majors versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-However it could happen that a new major version is desired while the active
-major version is still heavily used. Like what we had between php 4 and 5 or
-for the oldest, between php 3 and 4.
+Release Timeline
+================
 
-::
+The process starts the first Tuesday of July of each year, and nominally runs
+for 20 weeks. With 3 alpha releases, 3 beta releases, 4 release candidates,
+and a GA (x.0.0) release.
 
-    **** pre release phase
-    ++++ release lifetime bugs
-    ---- release lifetime security only
-    D    EOL
-    Version Time ->
-           2011        2012       2013         2014        2015        2016        2017
-            |     |     |     |     |     |     |     |     |     |     |     |     |
-    5.3     +++++++++++++-----D
-    5.4     |*****+++++++++++++++++++++++++-----------D     |     |     |     |     |
-    5.5     |     |     |******++++++++++++++++++++++++-----------D     |     |     |
-    5.6     |     |     |           |******++++++++++++++++++++++++-----------D
-    6.0     |     |     |******++++++++++++++++++++++++-----------D     |     |
-    6.1     |     |     |           |******++++++++++++++++++++++++-----------D
+Examples are given for 2024 and PHP 8.4. Releases are tagged on the Tuesday of
+each week, with a release before Thursday 24:00 (UTC).
+
+In the examples, `$rd` describes the release day of the first alpha release.
+
+Alpha Releases
+--------------
+
+.. list-table::
+   :header-rows: 0
+   :stub-columns: 1
+
+   * - Alpha 1
+     - * Tag on *First Tuesday of July*: ``$rd - 2`` (Jul 2, 2024)
+       * Release before *First Thursday of July*: ``$rd`` (Jul 4, 2024)
+   * - Alpha 2
+     - ``$rd + 14`` (Jul 18, 2024)
+   * - Alpha 3
+     - ``$rd + 28`` (Aug 01, 2024)
+
+During the alpha releases:
+
+- New features may be added at will, following the normal RFC procedures.
+
+Beta Releases
+-------------
+
+- Beta 1
+
+	- Tag / Feature Freeze: ``$rd + 40`` (Aug 13, 2024)
+	- Release: ``$rd + 42`` (Aug 15, 2024)
+
+- Beta 2: ``$rd + 56`` (Aug 29, 2024)
+- Beta 3: ``$rd + 70`` (Sep 12, 2024)
```

Comment:
My original suggestion for this got "outdated" by another change, so it was lost. Adding it again:

```suggestion
.. list-table::
   :header-rows: 0
   :stub-columns: 1
   * - Beta 1
     - * Tag / Feature Freeze: ``$rd + 40`` (Aug 13, 2024)
       * Release: ``$rd + 42`` (Aug 15, 2024)
   * - Beta 2
     - ``$rd + 56`` (Aug 29, 2024)
   * - Beta 3
     - ``$rd + 70`` (Sep 12, 2024)
```